src/app/settings/settings.component.ts

```diff
@@ -42,6 +42,10 @@ export class SettingsComponent implements OnInit {
       this.themeService.setTheme("dark");
       this.whichTheme = "dark";
       localStorage.setItem("theme", "dark");
+    } else if (this.selectedTheme == "iceydark") {
+      this.themeService.setTheme("iceydark");
+      this.whichTheme = "iceydark";
+      localStorage.setItem("theme", "iceydark");
```

Comment:
Instead of all these `if` blocks, can't we just simplify it to this:
```ts
this.themeService.setTheme(this.selectedTheme);
this.whichTheme = this.selectedTheme;
localStorage.setItem('theme', this.selectedTheme);
```